micro-rdk-nmea/src/messages/pgns.rs

```diff
@@ -126,7 +126,7 @@ pub struct GnssSatsInView {
 }
 
 macro_rules! define_pgns {
-    ( $(($pgndef:ident, $pgn:expr)),* ) => {
+    ( $($pgndef:ident),* ) => {
```

Comment:
I didn't push more for this change in #408 because I wasn't completely sure there wasn't a reason it didn't work, but it sure seems to.

Reply:
huh, ok. Sounds good to me

Reply:
I guess the one thing to check is whether this `NmeaMessageBody` type actually has any tests? If you want, I'll add one.